RollGen/Utils.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace RollGen
+{
+    public static class Utils
```

Comment:
Also, in line with the current solution, and given the comment above about hiding functionality, this should probably be marked internal.